hnn_core/network.py

```diff
@@ -1253,23 +1253,59 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
 
         self.connectivity.append(deepcopy(conn))
 
-    def clear_connectivity(self):
-        """Remove all connections defined in Network.connectivity
+    def clear_connectivity(self, src_types="all"):
+        """Remove connections with src_type in Network.connectivity.
+
+        Parameters
+        ----------
+        src_types : list | all | drives | local
```

Comment:
```suggestion
        src_types : list | 'all' | 'drives' | 'local'
```

to be clear that these are strings ... also fix below